server/templates/student/course/_assigntable.html

```diff
@@ -1,20 +1,20 @@
-{% macro render_assign(assignments, course, tname="Assignments", empty=False, scores=False) %}
+{% macro render_assign(assignments, course, tname="Assignments", empty=False, show_scores=False) %}
```

Comment:
This is probably better off as a separate issue: 

There are some courses that keep all of their assignments active (to allow late submissions I would guess). 
Since we are computing the scores anyway should we just show them somewhere - maybe under status? 
It'd be a bit weird if scores are in two different locations (for active vs inactive assignments).  
I can imagine having the score column on the active assignments becoming useful as we have continual autograding.